lighthouse-core/formatters/critical-request-chains.js

```diff
@@ -159,9 +161,14 @@ class CriticalRequestChains extends Formatter {
 
         // Create the appropriate tree marker based on the depth of this
         // node as well as whether or not it has children and is itself the last child.
-        const treeMarker = ancestorTreeMarker +
-            (isLastChild ? '┗━' : '┣━') +
+        let treeMarker = ancestorTreeMarker;
+        if (process.platform === 'win32') {
+          treeMarker += (isLastChild ? '\u2514\u2500' : '\u251C\u2500') +
+            (hasChildren ? '\u252C' : '\u2500');
```

Comment:
> This seems fine as a 2-space indentation

definitely needs a four space indent after a continuation like this :)

Happy to have a rule to enforce that...may just be the eslint config didn't bring that over when we updated it recently.